src/common/compiler_extensions.h

```diff
@@ -1,5 +1,5 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
-// vim: ts=8 sw=2 smarttab
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+// vim: ts=8 sw=2 expandtab
 /*
```

Comment:
Might suggest doing a find | egrep looking for files without the separating line, I won't keep commenting on them.  This is a nit of course.